src/main/kotlin/racing/model/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racing.model
+
+class Car {
+    private var moving: Int = 0
```

Comment:
가급적 `주생성자를 통해서 (객체의) 프로퍼티를 초기화`할 수 있도록 코드를 작성해보시면 어떨까요? 🙌

```suggestion
class Car(
    private var moving: Int = 0,
) {
```